assembly/std/storage.ts

```diff
@@ -263,13 +263,22 @@ export function getOf<T>(address: Address, key: T): T {
 }
 
 /**
- * Removes (key, value) from the datastore of the callee's address.
+ * Removes the key-value pair associated with the given `key` from the current contract's datastore.
  *
- * TODO: explains what happens on missing key.
- * TODO: explains security mechanisms
+ * @remarks
+ * If the key is not of type string, StaticArray<u8>, Args, or Uint8Array, an error will be thrown and
+ * the compilation will stop.
+ * If the caller is not authorized to delete the key-value pair, an error will be thrown.
+ * If the key does not exist an error will be thrown.
+ *
+ * @privateRemarks
+ * TODO: describe the security mechanisms involved in this operation.
```

Comment:
Should be in @remarks and should have an issue for the run team to come and explain this.

Reply:
Should I make one issue for each functions that need security mechanisms description. I already made a general one but I can transform it to meta issue

Reply:
Both are ok. But you should be explicite (list precisely all that need to be done)